src/sdk/pynni/nni/curvefitting_assessor/README.md

```diff
@@ -0,0 +1,68 @@
+* Step2: Predicting. Calculates the expected final result accuracy(implement by `f_comb`) at target position(ie the total number of epoch) by the \xi and the formula of the combined model.
+* Step3: If the fitting result doesn't converge, the predicted value will be `None`, in this case we return `AssessResult.Good` to ask for future accuracy information and predict again. Furthermore, we will get a positive value by `predict()` function, if this value is strictly greater than the best final performance in history * `THRESHOLD`(default value = 0.95), return `AssessResult.Good`, otherwise, return  `AssessResult.Bad`
+
+The figure below is the result of our algorithm on MNIST trial history data, where the green point represents the data obtained by Assessor, the blue point represents the future but unknown data, and the red line is the Curve predicted by the Curve fitting assessor.
+
+<p align="center">
+<img src="./example_of_curve_fitting.PNG" alt="drawing"/>
+</p>
+
+## 2. Usage
+To use Curve Fitting Assessor, you should add the following spec in your experiment's yaml config file:
+
+```
+assessor:
+    builtinAssessorName: Curvefitting
+    classArgs:
+      # (Require)The total number of epoch.
+      # We need to know the number of epoch to determine which point we need to predict.
+      epoch_num: 20
+      # (optional) choice: maximize, minimize
+      * The default value of optimize_mode is maximize
+      optimize_mode: maximize
+      # Kindly reminds that if you choose minimize mode, please adjust the value of threshold >= 1.0 (e.g threshold=1.1)
+      # (optional) A trial is determined to be stopped or not
+      # In order to save our computing resource, we start to predict when we have more than start_step(default=6) accuracy points.
+      # only after receiving start_step number of reported intermediate results.
+      * The default value of start_step is 6.
+      start_step: 6
+      # (optional) The threshold that we decide to early stop the worse performance curve.
+      # For example: if threshold = 0.95, optimize_mode = maximize, best performance in the history is 0.9, then we will stop the trial which predict value is lower than 0.95 * 0.9 = 0.855.
+      * The default value of threshold is 0.95.
+      shreshold: 0.95
+```
+
+## 3. File Structure
+The assessor has a lot of different files, functions and classes. Here we will only give most of those files a brief introduction:
+
+* `curvefunctions.py` includes all the function expression and default parameters.
+* `modelfactory.py` includes learning and predicting, the corresponding calculation part is also implemented here.
```

Comment:
also, shall we change to the name to "model_factory.py"? (just suggestion...)

Reply:
Modified, thanks for suggestions ：）